Extension/src/LanguageServer/client.ts

```diff
@@ -638,6 +645,17 @@ class DefaultClient implements Client {
         });
     }
 
+    public async logDiagnostics(): Promise<void> {
+        let response: GetDiagnosticsResult = await this.requestWhenReady(() => this.languageClient.sendRequest(GetDiagnosticsRequest, null));
+        if (!this.diagnosticsChannel) {
+            this.diagnosticsChannel = vscode.window.createOutputChannel("C/C++ Diagnostics");
+            this.disposables.push(this.diagnosticsChannel);
+        }
+        let header: string = `-------- Diagnostics - ${new Date().toLocaleString()}\n`;
+        let version: string = `Version: ${util.packageJson.version}\n`;
+        this.diagnosticsChannel.appendLine(`${header}${version}${response.diagnostics}`);
```

Comment:
Is there an API to "show" or "focus" the output channel to ensure that the user can see the output when we log it?